uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/filters/date-type.filter.js

```diff
@@ -33,18 +33,20 @@ define([
         if (nestedKeyDates) {
           _.forEach(nestedKeyDates, function (singleKeyDate) {
             if (dateTypeList.indexOf(singleKeyDate.type) !== -1) {
-              return filteredDateContactList.push(contactKeyDate);
+              filteredDateContactList.push(contactKeyDate);
+
+              return false; // break loop
             }
           });
         } else {
           if (dateTypeList.indexOf(contactKeyDate.type) !== -1) {
-            return filteredDateContactList.push(contactKeyDate);
+            filteredDateContactList.push(contactKeyDate);
```

Comment:
why `return` is removed?

Reply:
There is no need to return any value, as we are simply pushing the value to `filteredDateContactList`.